solidity/test/DepositRedemptionTest.js

```diff
@@ -340,16 +359,17 @@ describe("DepositRedemption", async function() {
       expect(events[0].returnValues.value).to.eq.BN(depositValue)
     })
 
-    it("escrows correct fee if deposit is pre-term, owner is TDT holder and signer fee is partially escrowed", async () => {
-      const expectedFee = new BN(100)
-      await tbtcToken.forceMint(testDeposit.address, signerFee.sub(expectedFee))
-      await feeRebateToken.transferFrom(accounts[4], owner, tdtId, {
-        from: accounts[4],
+    it("escrows signerFee if deposit is pre-term, owner is TDT holder and signer fee is partially escrowed", async () => {
```

Comment:
This will also need to be adjusted with the work we’re doing on #680.

Reply:
will adjust these with the relevant PR